rclcpp/test/rclcpp/test_graph_listener.cpp

```diff
@@ -170,26 +172,6 @@ TEST_F(TestGraphListener, error_run_graph_listener_mock_wait_set_add_guard_condi
     std::runtime_error("failed to add interrupt guard condition to wait set: error not set"));
 }
 
-TEST_F(TestGraphListener, error_run_graph_listener_mock_wait_set_add_guard_condition_twice) {
```

Comment:
this one was removed because it only failed when a second wait condition is added to the waitset, but the graphlistener is now only using one

Reply:
By the way, I don't like this kind of test that depends on an implementation detail (a guard condition being added twice to a waitset in the graph listener loop is not part of the API contract).
I'm not sure how to have 95% line coverage while avoiding this kind of tests though.